test/integration/submission/submission.spec.ts

```diff
@@ -850,8 +854,9 @@ describe('Submission Api', () => {
       } catch (err) {}
       await chai
         .request(app)
-        // data base is empty so ID shouldn't exist
-        .post('/submission/program/ABCD-EF/clinical/upload')
+        // database is empty so ID shouldn't exist
+        // .post('/submission/program/ABCD-EF/clinical/upload')
```

Comment:
If these commented lines are unused we should remove them

However, the bigger problem is that we now have two endpoints, `/upload` and `/submissionUpload`
If we deprecate `/upload` we have to make a change on the Gateway so existing functionality is not broken

And we should keep unit tests for `/upload` unless it is being removed... I'm wondering if we should we have separate unit tests for `/upload` and `/submissionUpload` at this stage

Reply:
Discussed on Slack -- plan to update after migration to Cumulus